README.md

```diff
@@ -69,26 +69,33 @@ Open the `.xcworkspace` file generated by `pod install` to access your project w
 #### Set up an application on Stitch
 1. Go to [https://stitch.mongodb.com/](https://stitch.mongodb.com/) and log in to MongoDB Atlas.
 2. Create a new app in your project with your desired name.
-3. Go to your app in Stitch via Atlas by clicking Stitch Apps in the left side pane and clicking your app.
-3. Copy your app's client app id by going to Clients on the left side pane and clicking copy on the App ID section.
-4. Go to Providers from Users in the left side pane and edit and enable "Allow users to log in anonymously".
+3. Go to your app in Stitch via Atlas by clicking 'Stitch' in the left side pane and clicking your app.
+3. Copy your Stitch App ID from the top of the left side pane.
+4. Enable anonymous authentication by going to the Users page in the left side pane, selecting the Providers tab, and enabling the "Allow users to log in anonymously" option.
 
-#### Set up a project in XCode/CocoaPods using Stitch
+#### Set up a project in Xcode/CocoaPods using Stitch
+
+1. Download and install [Xcode](https://developer.apple.com/xcode/). 
+
+- Note: Xcode 10.1 (10B61) is currently the maximum supported version. To install an old version of Xcode, see [https://developer.apple.com/download/more/](https://developer.apple.com/download/more/).
 
-1. Download and install [XCode](https://developer.apple.com/xcode/)
 2. Create a new app project with your desired name. Ensure that Swift is the selected language.
+
+- Note: iOS 11.0 is the minimum iOS Deployment Target.
+
 3. Navigate to the directory of the project in a command line, and run `pod init`.
 4. In the `Podfile` that is generated, add the following line under the dependencies for your app target:
 
 ```ruby
     pod 'StitchSDK', '= 5.1.0'
 ```
 
-5. Run `pod install`
-6. Open the generated `.xcworkspace` file
-7. Your app project will have all the necessary dependencies configured to communicate with MongoDB Stitch.
-8. To use basic Stitch features, `import StitchCore` in a source file.
-9. To access a remote MongoDB instance via Stitch, `import StitchRemoteMongoDBService` in a source file.
+See above for the list of optional Stitch service pods that you may add to your Podfile.
+
+5. Run `pod install`.
+6. Open the generated `.xcworkspace` file. Your app project will have all the necessary dependencies configured to communicate with MongoDB Stitch.
+7. To use basic Stitch features, `import StitchCore` in a source file.
```

Comment:
Might also be worth mentioning "To create BSON documents and BSON values like ObjectId, `import MongoSwift` in a source file"